migration/src/m0000840_add_relationship_14_15.rs

```diff
@@ -0,0 +1,45 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
```

Comment:
nit: make the data a global constant so you don't have to repeat it in 2 places.
```suggestion
const DATA: [(i32, &str); 2] = [(14, "DescribedBy"), (15, "PackageOf")];
```